### ISSUE TITLE
Help with binary bot error

### ISSUE THREAD

**Issue by fofuxoBR** [Fruitfella/ROBOT-binary#533]
Good night, I tried to follow the tutorial in this video:
https://www.youtube.com/watch?v=EnQOcxEn8tg
but at the time of the test it presents the following error:
undefined is not a function. I've done everything to solve it and nothing, can anyone help?

send the file created for analysis.

ps: I'm Brazilian, so:

Starting bet = Stake

Bet after win = Stake Win

How much do you want to earn? = Take profit

Maximum loss = loss

link to donwload: 
https://www.mediafire.com/file/wddxh8a4d9cg0ne/TESTE+(1).xml/file

thank you.

**Comment by h3ndr4p1n** [comment]
[CROSS-MAE-7-21.zip](https://github.com/Fruitfella/ROBOT-binary/files/5054695/CROSS-MAE-7-21.zip)
here try this

**Comment by fofuxoBR** [comment]
password?

**Comment by h3ndr4p1n** [comment]
password

**Comment by tymu** [comment]
https://www.youtube.com/watch?v=NqLGLvv1Ceo&ab_channel=TesterT
